src/Microsoft.AspNet.OData/GetNextPageHelper.cs

```diff
@@ -13,12 +13,12 @@ namespace Microsoft.AspNet.OData
     internal static partial class GetNextPageHelper
     {
         /// <remarks>This signature uses types that are AspNet-specific.</remarks>
-        internal static Uri GetNextPageLink(Uri requestUri, int pageSize)
+        internal static Uri GetNextPageLink(Uri requestUri, int pageSize, object instance = null, Func<object, string> objectToSkipTokenValue = null)
         {
             Contract.Assert(requestUri != null);
             Contract.Assert(requestUri.IsAbsoluteUri);
```

Comment:
please remove this assert; no longer valid.

Reply:
still appears to be there.

---
In reply to: [287530965](https://github.com/OData/WebApi/pull/1724#discussion_r287530965) [](ancestors = 287530965)